dev-libs/opensp/opensp-1.5.2.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="A C++ library for using SGML/XML files"
+DESCRIPTION="An SGML Extended Facilities system conforming to Annex A of International Standard ISO/IEC 10744 -- Hypermedia/Time-based Structuring Language"
+HOMEPAGE="https://openjade.sourceforge.net/"
+COPYRIGHT="1994, 1995, 1996, 1997, 1998 James Clark"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/openjade/OpenSP-$portVersion.tar.gz"
+CHECKSUM_SHA256="57f4898498a368918b0d49c826aa434bb5b703d2c3b169beb348016ab25617ce"
+SOURCE_DIR="OpenSP-$portVersion"
+PATCHES="opensp-1.5.2.patchset"
+
+ARCHITECTURES="all"
+
+libVersion="5.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	opensp = $portVersion
+	cmd:onsgmls = $portVersion
+	cmd:osgmlnorm = $portVersion
+	cmd:ospam = $portVersion
+	cmd:ospcat = $portVersion
+	cmd:ospent = $portVersion
+	cmd:osx = $portVersion
+	lib:libosp = $libVersionCompat
+	"
+
+REQUIRES="
+	haiku
+	"
+
+PROVIDES_devel="
+	opensp_devel = $portVersion
+	devel:libosp = $libVersionCompat
+	"
+
+REQUIRES_devel="
+	haiku_devel
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+BUILD_PREREQUIRES="
+	cmd:automake
+	cmd:autoreconf
+	cmd:gcc
+	cmd:gettext
+	cmd:libtoolize
+	cmd:make
+	cmd:perl
+	"
+
+BUILD()
+{
+	autoreconf -if
+	runConfigure ./configure --disable-doc-build
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make PREFIX=$prefix install
+	mkdir -p $developLibDir
+	rm $libDir/lib*.la
+	mv $libDir/lib*.a $developLibDir
```

Comment:
What are these static libraries?

Reply:
Didn't check here, but I guess the static one could be disabled while running configure with the option `--disable-static`?

Reply:
The next library in chain, libofx (I haven't uploaded the recipe yet), needs the *.a file, so apparently links static against opensp. Anything we should do about it?

Reply:
If it isn't needed by anything else, maybe we can only the static library?

Reply:
I commited a new version, only the static library is kept. In my point of view, ideally we should be able to keep the SO library in the opensp package, and the static (.a) in opensp_devel, but prepareInstalledDevelLib doesn't like it. Should we avoid running prepareInstalledDevelLib, split the packages somehow, delete the shared library as I'm doing here?

Reply:
IIRC static libraries are to be avoided if not strictly needed, can't have both though.

Reply:
Checked build on 32bit, static library not needed, so this should be disabled with "--disable-static"